src/main/scala/backends/futil/FutilAst.scala

```diff
@@ -148,6 +164,18 @@ object Futil {
     }
   }
 
+  sealed trait GuardExpr extends Emitable {
+    override def doc(): Doc = this match {
+      case Atom(item) => item.doc
+      case And(left, right) => left.doc <+> text("&") <+> right.doc
+      case Or(left, right) => left.doc <+> text("|") <+> right.doc
```

Comment:
Do you not need to parenthesize this?

Reply:
Nope, this was just an oversight on my part. thanks for catching this!